.github/workflows/pull-request-target.yml

```
@@ -10,9 +10,13 @@
 ##############################################################################
 
 name: PR Target Workflow
-on: 
+on:
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)